File: src/java/src/main/java/com/cecd/sdk/rpc/authcenter/Libraries/AccountLib.java
package com.cecd.sdk.rpc.authcenter.Libraries;

import com.alibaba.fastjson.JSON;
import com.alibaba.fastjson.JSONObject;
import com.cecd.sdk.rpc.RpcDoc;
import com.cecd.sdk.rpc.RpcInterface;

/**
 * 这里编写暴露的rpc客户端
 *
 */
@RpcDoc("App\\Libraries\\AccountLib")
public interface AccountLib extends RpcInterface {
    public JSONObject check(String accessToken);
}

File: src/php/example/AuthCenter/Libraries/AccountLib.php
<?php
/**
 * Created by PhpStorm.
 * User: zhangyubo
 * Date: 2019/4/25
 * Time: 11:39
 */

namespace Cecd\Sdk\Rpc\AuthCenter\Libraries;

use Thrift\CeCd\Sdk\Core\Services\RpcInterface;

/**
 * Interface AccountLib
 * @package Cecd\Sdk\Rpc\AuthCenter\Libraries
 * @classpath(\App\Libraries\AccountLib)
 */
interface AccountLib extends RpcInterface
{
    public function check(string  $token) : array ;
}